### PR TITLE
bsunanda:Run2-alca67 Changes needed for IsoTrack validation (Data vs MC)

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/IsolatedParticles/plugins/StudyHLT.cc [hunk 1]
```diff
@@ -1,35 +1,137 @@
...
+#include "TH2.h"
+#include "TTree.h"
 
+// user include files
 #include "Calibration/IsolatedParticles/interface/FindCaloHit.h"
 #include "Calibration/IsolatedParticles/interface/CaloPropagateTrack.h"
 #include "Calibration/IsolatedParticles/interface/ChargeIsolation.h"
 #include "Calibration/IsolatedParticles/interface/eECALMatrix.h"
 #include "Calibration/IsolatedParticles/interface/eHCALMatrix.h"
+#include "Calibration/IsolatedParticles/interface/TrackSelection.h"
```
Comment (mmusich) [inline Calibration/IsolatedParticles/plugins/StudyHLT.cc]: @bsunanda are really all of these new includes needed?

Comment (bsunanda) [inline Calibration/IsolatedParticles/plugins/StudyHLT.cc]: @mmusich They are indeed needed - they have been there divided into .h and .cc files. Now they are combined into one (I have removed the .h file).

Comment (mmusich) [inline Calibration/IsolatedParticles/plugins/StudyHLT.cc]: I see, thanks

**Comment by mmusich** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**